test/controllers/DetermineEligibilityControllerSpec.scala

```diff
@@ -34,64 +35,92 @@ class DetermineEligibilityControllerSpec extends ItSpec {
 
   "Determine eligibility endpoint should route user correctly and send an audit event" - {
     forAll(Table(
-      ("Scenario flavour", "eligibility rules", "ineligibility reason audit string", "expected redirect", "updated journey json"),
+      ("Scenario flavour", "eligibility rules", "ineligibility reason audit string", "expected redirect", "updated journey json", "tax regime"),
```

Comment:
tiny thing, but the new field isn't taxRegime, but origin instead

Reply:
oh yeah, good spot - I"ll change it